lib/alice/handlers/karma.ex

```diff
@@ -46,11 +53,15 @@ defmodule Alice.Handlers.Karma do
     |> reply("#{term} has had its karma scattered to the winds. (#{count})")
   end
 
+  defp increment(term, term, conn), do: reply(conn, ~s(#{@karma_denial_message}))
```

Comment:
Isn't the 2nd parameter a `user`?

Reply:
It is but we're using pattern matching here to say if the `user` that is passed along is equal to `term` then call `reply..`

Reply:
they should both be `user` then

Reply:
sorry i meant, we're using pattern matching here to say if the `term` that is getting it's karma incremented is the same as the `user`, then reply with the denial message